man/man8/zfs-allow.8

```diff
@@ -213,6 +213,7 @@ release	subcommand	Allows releasing a user hold which might destroy the snapshot
 rename	subcommand	Must also have the \fBmount\fR and \fBcreate\fR ability in the new parent
 rollback	subcommand	Must also have the \fBmount\fR ability
 send	subcommand
+send:encrypted        subcommand      Allows a user to send encrypted datasets as a raw datastream. Restricts a user from sending an dencrypted dataset.
```

Comment:
While there, couldn't we also comment the traditional send for comparison?  I am not sure "Restrict" part is right here, since I suppose if both "send" and "send:encrypted" are set, then first should win.

Reply:
And there is a typo there too